bindings/node/.evergreen/setup_environment.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+NODE_LTS_NAME=${NODE_LTS_NAME:-dubnium}
```

Comment:
just to clarify, we're only testing `dubnium` right now, correct?

Reply:
We are only testing dubnium, but we are still building all node version prebuilds from dubnium in #39 . I was a little concerned about overwhelming the testing matrix with node variants.